examples/examples-release-11/src/main/resources/application.properties

```diff
@@ -1,2 +1,3 @@
 namespace=airflow
-management.endpoints.web.exposure.include=prometheus
\ No newline at end of file
+management.endpoints.web.exposure.include=prometheus
+kubernetes.actuator.prometheus.enabled=true
```

Comment:
I think this property is no longer used.

Reply:
thanks, removed